lib/galaxy/jobs/runners/univa.py

```diff
@@ -86,21 +79,21 @@ def _complete_terminal_job(self, ajs, drmaa_state, **kwargs):
             mem_wasted = extinfo["memory_wasted"]
             slots = extinfo["slots"]
 
-            # log.debug("UnivaJobRunner:_complete_terminal_job ({jobid}) memviolation {mv}".format(jobid=ajs.job_id, mv=memviolation))
+                # log.debug("UnivaJobRunner:_complete_terminal_job ({jobid}) memviolation {mv}".format(jobid=ajs.job_id, mv=memviolation))
```

Comment:
Can you just drop the comment ?
Else you need to drop the extra indentation here.

Reply:
Will drop the extra indentation. If debugging is necessary its easier to renable. I guess that some debugging might be necessary if others start using it. 

Will do it everywhere.